modules/ballerina-core/src/main/java/org/ballerinalang/model/statements/StatementType.java

```diff
@@ -0,0 +1,73 @@
+/*
+*   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.ballerinalang.model.statements;
+
+/**
+ * Represents Ballerina Statement Type.
+ *
+ * @since 0.89
+ */
+public enum StatementType {
```

Comment:
I think we need to find a better classname here. This class contain both blocks and statements